drizzle/migrations/0003_add_clerk_auth_helpers.sql

```
@@ -0,0 +1,166 @@
+CREATE POLICY "Only owners can delete participants" ON public.chat_participants
+FOR DELETE USING (
+  EXISTS (
+    SELECT 1 FROM public.chat_participants cp
+    WHERE cp.chat_id = chat_participants.chat_id
+    AND (cp.user_id = auth.uid() OR public.is_clerk_user(cp.user_id))
+    AND cp.role = 'owner'
+  )
+);
+--> statement-breakpoint
+
+DROP POLICY IF EXISTS "Only owners can insert participants" ON public.chat_participants;
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)